src/platform/plugins/shared/files/server/routes/common_schemas.ts

```diff
@@ -9,38 +9,20 @@
 
 import { schema, Type } from '@kbn/config-schema';
 
-const ALPHA_NUMERIC_WITH_SPACES_REGEX = /^[a-z0-9\s_]+$/i;
-const ALPHA_NUMERIC_WITH_SPACES_EXT_REGEX = /^[a-z0-9\s\._]+$/i;
-
-function alphanumericValidation(v: string) {
-  return ALPHA_NUMERIC_WITH_SPACES_REGEX.test(v)
-    ? undefined
-    : 'Only alphanumeric characters are allowed as file names';
-}
-
-function alphanumericWithExtValidation(v: string) {
-  return ALPHA_NUMERIC_WITH_SPACES_EXT_REGEX.test(v)
-    ? undefined
-    : 'Only alphanumeric characters, spaces (" "), dots (".") and underscores ("_") are allowed';
-}
-
 export const fileName = schema.string({
   minLength: 1,
   maxLength: 256,
-  validate: alphanumericValidation,
```

Comment:
The change looks good to me :D, just want to double check

@jloleysens, any chance you remember if there was a good reason for this additional validation?

Reply:
The idea was just to prevent crazy unicode names. Setting a contract for what we consider "valid" names seemed like the easiest way to do that.

I can see how this limitation does not work with languages that need multibyte input so removing it makes sense, but fwiw that means [this issue](https://github.com/elastic/kibana/issues/210312) is not actually a bug, it's the original design :D